site/docs/4.5/components/toasts.md

```diff
@@ -167,7 +167,7 @@ You can also get fancy with flexbox utilities to align toasts horizontally and/o
 
 {% capture example %}
 <!-- Flexbox container for aligning the toasts -->
-<div aria-live="polite" aria-atomic="true" class="d-flex justify-content-center align-items-center" style="min-height: 200px;">
+<div aria-live="polite" aria-atomic="true" class="d-flex justify-content-center align-items-center" style="height: 200px;">
```

Comment:
Why change the `min-height` to `height`?

Reply:
This is the 2. in the description, should have be clearer: flexbugs #3 concerns IE11 that doesn't justify content correctly when the flex container only has a `min-height`.

The centered example was top-aligned in IE11. Since I don't think it has any bad consequence, I simply switched to `height` 🙂